packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -552,6 +556,14 @@ export function TokenSearch({
           l2Address: _token.l2Address
         })
       }
+
+      if (
+        isWithdrawOnlyToken(_token.address, l2.network.id) ||
+        isTransferDisabledToken(_token.address, l2.network.id)
+      ) {
+        openWithdrawOnlyDialog()
```

Comment:
NIT: If we are using the same dialog for `isWithdrawOnlyToken` and `isTransferDisabledToken` then I think it's better we rename the dialog to `openTransferDisabledDialog` (and change other variable names correspondingly) because just being named after withdrawal is misleading.